src/containers/User/SettingsProfile.js

```diff
@@ -64,8 +53,23 @@ export const SettingsProfile = props => {
       .then((response) => {
         const {statusCode, data} = response
         if (statusCode == 200) {
-          Alert.alert(t('You\'re profile has been updated'))
+          Alert.alert(t('Your profile has been updated'))
           setSubmitPending(false)
+          // update profile context if username or user_image changed
+          if (values['user_image'] &&
```

Comment:
maybe a little bit more readable?

`if (values?.user_image !== profileContext?.userImage) {..}`

not sure, leave it if its more confusing

Reply:
But that wouldn't work? The first condition is to check if there is user_image in the values and if I got you right `values?.user_image` will tell me if values exist? And I think if it doesn't, it will give `undefined` which will be `!== profileContext?.userImage`, which was also the peoblem why the condition above with the images didn't work and gave a lot of bad requests when we only changed the username and not the image.

Reply:
yes you are right! ..sorry!